metricbeat/etc/fields.yml

```diff
@@ -205,6 +205,343 @@
+                  description: >
+                    The number of connections to the database server from clients. This
+                    number includes the current shell session. Consider the value of
+                    available to add more context to this datum.
+                - name: available
+                  type: long
+                  description: >
+                     The number of unused available incoming connections the database
+                     can provide.
+                - name: total_created
+                  type: long
+                  description: >
+                    A count of all incoming connections created to the server. This
+                    number includes connections that have since closed.
+
+            - name: journaling
+              type: group
+              description: >
+                Data about the journaling-related operations and performance. Journaling
+                information only appears for mongod instances that use the MMAPv1
+                storage engine and have journaling enabled.
+              fields:
+                - name: commits
+                  type: long
+                  description: >
+                    The number of transactions written to the journal during the last
+                    journal group commit interval.
+                - name: journaled.mb
+                  type: long
+                  description: >
+                    The amount of data in megabytes (MB) written to journal during the
+                    last journal group commit interval.
+                - name: write_to_data_files.mb
+                  type: long
+                  description: >
+                    The amount of data in megabytes (MB) written from journal to the
+                    data files during the last journal group commit interval.
+                - name: compression
+                  type: long
+                  description: >
+                    The compression ratio of the data written to the journal.
+                - name: commits_in_write_lock
+                  type: long
+                  description: >
+                    Count of the commits that occurred while a write lock was held.
+                    Commits in a write lock indicate a MongoDB node under a heavy write
+                    load and call for further diagnosis.
+                - name: early_commits
+                  type: long
+                  description: >
+                    The number of times MongoDB requested a commit before the scheduled
+                    journal group commit interval.
+                - name: times
+                  type: group
+                  description: >
+                    Information about the performance of the mongod instance during the
+                    various phases of journaling in the last journal group commit
+                    interval.
+                  fields:
+                    - name: dt.ms
```

Comment:
Hmm, not sure, I think "delta t". I could expand it to `delta_t`, not sure if that would be better.